railties/test/fixtures/.devcontainer/compose.yaml

```diff
@@ -0,0 +1,45 @@
+services:
+  rails-app:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+
+    volumes:
+      - ../..:/workspaces:cached
+
+    # Overrides default command so things don't shut down after the process ends.
+    command: sleep infinity
+
+    networks:
+    - default
+
+    # Uncomment the next line to use a non-root user for all processes.
+    # user: vscode
+
+    # Use "forwardPorts" in **devcontainer.json** to forward an app port locally.
+    # (Adding the "ports" property to this file will not forward from a Codespace.)
+    ports:
+    - 45678:45678
```

Comment:
If you want that, feel free to change in your app. Rails by design binds the development environment to 0.0.0.0.

Reply:
Although this might not be necessary. We are exposing the port so the app container talks with the selenium container. If we use the bridge mode that should not be necessary. I'll investigate.

Reply:
Will be fixed in https://github.com/rails/rails/pull/52020. Thank you for pointing it.